fdbclient/ClientKnobCollection.h

```diff
@@ -18,6 +18,9 @@
  * limitations under the License.
  */
 
+#ifndef FDBCLIENT_CLIENTKNOBCOLLECTION_H
```

Comment:
These header guards are unnecessary since we are using `#pragma once`.

Reply:
I think this is a good change, because `#pragma once` is a non-standard, c.f. https://en.wikipedia.org/wiki/Pragma_once